package.json

```diff
@@ -31,7 +31,7 @@
     "ethereumjs-util": "^7.1.4",
     "ethers": "^5.5.4",
     "hardhat": "npm:@celo/hardhat@2.8.4",
-    "hardhat-deploy": "^0.10.5",
+    "@pahor167/hardhat-deploy": "^0.11.19",
```

Comment:
Create this as a repo under the celo-org, use that instead.

Don't even necessarily need to release it on NPM, you should be able to specify a github repo as a package source in `package.json`.

Reply:
changed to github repo